editoast/src/modelsv2/scenario.rs

```diff
@@ -0,0 +1,126 @@
+use crate::error::Result;
+use crate::models::List;
+use crate::modelsv2::{Model, Ordering, Row};
+use crate::schema::v2::scenario::{ScenarioV2, ScenarioWithDetailsV2};
+use crate::views::pagination::{Paginate, PaginatedResponse};
+use async_trait::async_trait;
+use chrono::NaiveDateTime;
+use diesel::sql_query;
+use diesel::sql_types::{BigInt, Text};
+use diesel::{ExpressionMethods, QueryDsl};
+use diesel_async::{AsyncPgConnection as PgConnection, RunQueryDsl};
+use editoast_derive::ModelV2;
+use serde_derive::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, ModelV2, Deserialize, Serialize)]
+#[model(table = crate::tables::scenario_v2)]
+#[model(changeset(derive(Debug), public))]
+pub struct ScenarioModel {
```

Comment:
```suggestion
pub struct Scenario {
```

Reply:
Perhaps we should establish a naming convention for this; I've observed that we currently employ `*Model` for structures that access the database, while using only the entity's name for OpenAPI schemas.